JitsiConference.js

```diff
@@ -1123,7 +1123,12 @@ JitsiConference.prototype.addTrack = function(track) {
         return Promise.reject(new Error(`Cannot add second ${mediaType} track to the conference`));
     }
 
-    return this.replaceTrack(null, track);
+    return this.replaceTrack(null, track)
+        .then(() => {
+            if (track.getVideoType() === VideoType.DESKTOP && FeatureFlags.isMultiStreamSupportEnabled()) {
```

Comment:
can you add a comment which explains that it's needed for desktop, because camera is the default video type? but then I'm wondering if the presence does not need to be updated anyway to send the "unmuted" status.

Reply:
The videoType is what matters here more than the unmuted status. If the signaling reaches the peer before the presence does, then Jitsi-meet won't create a fake participant tile since the default videoType is camera.

Reply:
I will add a comment

Reply:
hmm okay... I hope at some point we'll be able to remove the video type from presence, freeze it (disallow video type changes) and send it over the signaling only. Then this could go away